src/NHibernate.Test/CfgTest/AccessorsSerializableTest.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace NHibernate.Test.CfgTest
 {
+	[TestFixture]
 	public class AccessorsSerializableTest
```

Comment:
Async counterparts are currently generated only for classes marked with `TestFixture`. Lacking it cause the fixture to be ignored. I have added the attribute on all fixture defining tests and lacking it.

Since this attribute is no more needed by NUnit, we are effectively using it as a kind of `GenerateAsync` attribute...

We could add our own specific attribute for that and replace all `TextFixture` by it. It would be more explicit.